Source/Core/Core/WiiRoot.cpp

```diff
@@ -32,6 +34,48 @@ namespace FS = IOS::HLE::FS;
 
 static std::string s_temp_wii_root;
 
+static std::string FileNameFromPath(const std::string& path)
+{
+  std::string file_name, extension;
+  SplitPath(path, nullptr, &file_name, &extension);
+  return file_name + extension;
+}
+
+static bool CopyBackupFile(const std::string& path_from, const std::string& path_to)
```

Comment:
Why is this required? File::Copy should already fail if the source file doesn't exist. And the destination file is overwritten if it already exists (bFailIfExists=false is passed to [CopyFile](https://docs.microsoft.com/en-us/windows/win32/api/winbase/nf-winbase-copyfile) and ofstream is opened in non-append mode)

Reply:
Ah, I didn't know it would be overwritten if it exists. As for the check for whether the source file exists, I thought it would be nice to avoid getting ERROR_LOGs on every normal boot of Dolphin. I can remove the target deletion, but do you think I should remove the source existence check as well?

Reply:
Ah, that makes sense. I agree with keeping the existence check then.